lms/static/sass/shared/_footer-edx.scss

```diff
@@ -95,6 +199,7 @@ footer#footer-edx-v3 {
     @include margin-left(8px);
 
     border-bottom: 1px solid $courseware-button-border-color;
+    line-height: 24px;
```

Comment:
```line-height``` is usually set in relation to font-size. So if the font-size is 16px this could be set as ```line-height: 1.5```.